src/redux/countriesSlice.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Kindly remove this eslint disable rule it's best to fix the issue in the code.

Reply:
Thank you @mikenath223
I have resolved this issue